spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/data/parquet/vectorized/TestParquetVectorizedReads.java

```diff
@@ -139,6 +164,19 @@ FileAppender<GenericData.Record> getParquetWriter(Schema schema, File testFile)
     return Parquet.write(Files.localOutput(testFile)).schema(schema).named("test").build();
   }
 
+  FileAppender<GenericData.Record> getEncryptedParquetWriter(Schema schema, File testFile)
```

Comment:
Iceberg methods should not use `get` in names.

Reply:
this is named after the existing `getParquetWriter` and `getParquetV2Writer` methods. Will change all in this class.